Inc/Data.h

```diff
@@ -24,6 +24,12 @@ typedef struct
     int32_t temperature_;
 } BarometerData;
 
+typedef struct
+{
+    osMutexId mutex_;
+    int32_t pressure_;
+} CombustionTankPressureData;
```

Comment:
Not really a tank since the bottom is open, it's a chamber, but it's ok, not really important.